tests/components/deconz/conftest.py

```diff
@@ -19,9 +19,14 @@
 from tests.components.light.conftest import mock_light_profiles  # noqa: F401
 from tests.test_util.aiohttp import AiohttpClientMocker
 
-type ConfigEntryFactoryType = Callable[
-    [MockConfigEntry], Coroutine[Any, Any, MockConfigEntry]
-]
+
+class ConfigEntryFactoryType(Protocol):
+    """Fixture factory that can set up deCONZ config entry."""
+
+    async def __call__(self, x: MockConfigEntry = ...) -> MockConfigEntry:
```

Comment:
```suggestion
    async def __call__(self, entry: MockConfigEntry = ...) -> MockConfigEntry:
```

The argument name should match that of `__mock_setup_config_entry`.

_Thinking a bit further, it might make sense to make `entry` a positional only argument here as well by added `/` after `= ...`. Update `__mock_setup_config_entry` in that case as well._